app/models/time_profile.rb

```diff
@@ -187,9 +172,26 @@ def self.profile_for_user_tz(user_id, user_tz)
 
   # @param tz [nil|TimeProfile|TimeZone] (default timezone "UTC")
   # @return [TimeProfile] time profile that uses this time zone
-  def self.default_time_profile(tz = DEFAULT_TZ)
-    return tz if tz.kind_of?(TimeProfile)
-    tz ||= DEFAULT_TZ
-    rollup_daily_metrics.find_all_with_entire_tz.detect { |tp| tp.tz_or_default == tz }
+  def self.default_time_profile(tzone = DEFAULT_TZ)
```

Comment:
Also don't like this change...`tz` is a fine variable name in context

Reply:
Restored.